frontend/src/ui/pages/RegisterPage/RegisterPage.scss

```diff
@@ -0,0 +1,139 @@
+@import '../../../colors.scss';
+@import url('https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap');
+
+.register-page {
+    display: flex;
+    align-items: center;
+    gap: 7rem;
+    justify-content: center;
+    // margin-top: -5rem;
+}
+
+.register-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    // margin-top: 5rem;
+}
+
+.register-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 481px;
+    gap: 15px;
+}
+
+.header {
+    background-color: $button-background;
+    border: 2px black solid;
+    padding: 10px 0;
+    width: 100%;
+    text-align: center;
+    color: $button-text;
+    font-size: 2rem;
+    margin: 0;
+}
+
+.register-field-box {
+    width: 100%;
+}
+
+.register-field {
+    padding: 0;
+    padding-block: 0;   
+    padding-inline: 0; 
+    font-size: 32px;
+    text-align: center;
+    width: 100%;
+    height: 89px;
+    margin-left: -2px;
+    border: 2px black solid;
+    &::placeholder {
+        font-family: var(--font-press-start-2p);
+        font-size: 20px;
+        color: rgba(142, 142, 142, 0.79);
+    }
+}
+
+.error-msg {
+    font-size: 12px;
+    margin-top: 15px;
+    text-align: center;
+}
+
+.btn {
+    border: none;
+    width: 100%;
+    // height: 124px;
+    font-size: 10px;
+    // padding: 10px 0;
+    background-color: $button-start !important;
+    font-size: 1.3rem;
```

Comment:
Many instances of the same CSS property are being overwritten within the same selector (e.g., font-size here). Could you please review the entire CSS files in this regard?